test/test_helper.rb

```diff
@@ -5,12 +5,15 @@
 
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'ttnt'
+require 'helpers/git_helper'
+require 'helpers/rake_helper'
 
+require 'rugged'
 require 'minitest/autorun'
 
 module TTNT
   class TestCase < Minitest::Test
-    FIZZBUZZ_FIXTURE_DIR = File.expand_path('../fixtures/repositories/fizzbuzz', __FILE__).freeze
+    FIXTURE_DIR = File.expand_path('../fixtures', __FILE__).freeze
```

Comment:
Oh, that's right, thank you! Fixed here: https://github.com/Genki-S/ttnt/commit/5647d9785c3c40126f86a8654f2663ae2a1b95d4